widgetastic_manageiq/__init__.py

```diff
@@ -4684,6 +4704,7 @@ class MigrationProgressBar(Widget):
     SIZE_LOCATOR = './/strong[contains(@id,"size-migrated")]'
     # TODO: XPATH requested to devel (repo:miq_v2v_ui_plugin issues:415)
     VMS_LOCATOR = './/strong[contains(@id,"vms-migrated")]'
+    PROGRESS_BAR = 'div[contains(@class,"progress")]/div[contains(@class,"in-progress")]'
```

Comment:
REQUIRED: Always start your xpath locators with . to ensure they are not looked up on whole page but just only in the context of the container.

Reply:
@sshveta, agreed. corrected.